ledger/participant-integration-api/src/main/scala/platform/store/backend/common/CommonStorageBackend.scala

```diff
@@ -524,6 +525,32 @@ private[backend] trait CommonStorageBackend[DB_BATCH] extends StorageBackend[DB_
             )"""
```

Comment:
This Event pruning would be probably more well suited in the EventStorageBackendTemplate, especiallly since it using templates now. (if you are under time pressure never mind, we can do it in the cleanup epic)

Reply:
That's a good point. I'll add a TODO to it and please let's leave it for the cleanup epic.